packages/preview/hydra/0.2.0/typst.toml

```
@@ -0,0 +1,11 @@
+[package]
+entrypoint = "src/lib.typ"
+name = "hydra"
+version = "0.2.0"
+compiler = "0.8.0"
+authors = ["tinger <me@tinger.dev>"]
+repository = "https://github.com/tingerrr/hydra"
+description = "Query and display headings of the currently active section"
+keywords = ["heading", "introspection"]
+license = "MIT"
+exclude = ["example", "README.md"]
```

```suggestion
exclude = ["examples", "README.md"]
```
seems to be correct in 0.1.0, I wonder why this changed

---

I must've forgotten to change it in the repo itself.